packages/zosjobs/src/doc/input/IDownloadSpoolContentParms.ts

```diff
@@ -49,4 +49,14 @@ export interface IDownloadSpoolContentParms extends Omit<IDownloadAllSpoolConten
      * @memberof IDownloadAllSpoolContentParms
      */
     stream?: Writable;
+
+    /**
+     * Wait for the job to reach output status
+     */
+    waitForActive?: boolean;
+
+    /**
+     * Wait for the job to reach output status
+     */
+    waitForOutput?: boolean;
```

Comment:
Adding the properties to this file shouldn't be needed, since `IDownloadSpoolContentParms` extends the interface `IDownloadAllSpoolContentParms`.